src/views/Package/components/OperatorArea/OperatorArea.tsx

```diff
@@ -39,12 +40,14 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
     }
 
     if (username) {
-      const managedBy = (
+      const managedBy = authorUrl ? (
```

Comment:
```suggestion
      const managedBy = author ? (
```

Reply:
Not sure I follow. What is `author`?

Reply:
Perhaps `const author = ...`, we still want to check if a URL exists to determine if we render a link or text

Reply:
Going to assume that yes thats what you meant - changed.